simcon_project/users/models/researcher.py

```diff
@@ -1,9 +1,7 @@
 from django.db import models
-from conversation_templates.models.conv_template import ConversationTemplate
+from .custom_user import CustomUser
 
 
 class Researcher(CustomUser):
-    templates = models.ForeignKey(ConversationTemplate, on_delete=models.CASCADE)
```

Comment:
Why have we gotten rid of all these fields?

Reply:
we found out that the way we had some of the relationships set up was wrong so we moved things around. It ended up that researcher doesn't have any unique fields but we left the class in case we want to add something to it later.

Reply:
Okay, that sounds good